views/components/GraphTable/index.jsx

```diff
@@ -48,10 +41,14 @@ const GraphTable = ({ graph, classes }) => {
         <TableBody>
           {tableRows.map((row, index) => {
             return (
-              <TableRow hover key={tableColumns[index] + index}>
+              <TableRow hover key={tableColumns[index] + '-' + index}>
                 {row.map(({ data, color }, i) => {
                   return (
-                    <TableCell align="center" style={{ color }} key={i}>
+                    <TableCell
+                      align="center"
+                      style={{ color }}
+                      key={data + '-' + i + '-' + color}
```

Comment:
Total nit but we could use a template string here: 
```suggestion
                      key={`${data}-${I}-${color}`}
```